playbooks/common/openshift-etcd/migrate.yml

```diff
@@ -3,6 +3,20 @@
   tags:
   - always
 
+- name: Check masters for embedded etcd
+  hosts: localhost
+  connection: local
+  become: no
+  gather_facts: no
+  tags:
+  - always
+  tasks:
+  - fail:
+      msg: "Migration of clustered embedded etcd instances is not supported. Currently there are {{ groups.oo_masters_to_config | length }} masters."
```

Comment:
nit: This error message would confuse me. Maybe add something like "There can only be one $deployment_type master when migrating etcd. Currently there.."